### ISSUE TITLE
Tabs not fitting in bar

### ISSUE THREAD

**Issue by jmbmkn** [Zonnev/elementaryos-firefox-theme#141]
![image](https://user-images.githubusercontent.com/402149/130369111-877d9500-35aa-4f4b-95d4-f086b48f9c41.png)
Looks like the tab is too big for the tab bar. I have a hiDPI Screen if that might be the cause.

**Comment by Zonnev** [comment]
Setting hiDPI doesn't change the tabs height so nothing changes. In your screen capture the tab looks bigger in height than in my theme. I don't know why that is. Maybe it's an extension for the tabs that you use? My theme doesn't support extensions.

**Comment by jmbmkn** [comment]
I've tried disabling all extensions and it's still the same. Is there an easy way to test a 'clean' firefox?

**Comment by Blast-City** [comment]
Hi.

Use the command "**firefox -p**" on terminal and create a new profile.

**Comment by jmbmkn** [comment]
I tried a new profile and could not get the theme to apply on the new profile. I got very confused and deleted a lot of firefox folders :confused: Anyway, I needed to reinstall elementary when the latest build came out (Dell UEFI issue). So when I reinstalled Firefox from flathub and signed in I already had all my extensions disabled. I enabled them one by one and this issue with the big tab bar has not returned. I don't know if it was the reinstall of eOS resolved it, but it's no longer an issue so this can be closed and I'll update if I am able to reproduce it.
